idemp-rest-test/testing/src/api/index.ts

```diff
@@ -17,4 +17,5 @@ export * from './products';
 export * from './referenceLists';
 export * from './visits';
 export * from './warehouses';
-
+export * from './receiveProducts';
```

Comment:
Will you run `npm run create-index` in this project? It will sort this file for you.

Reply:
Done